README.md

```diff
@@ -67,28 +67,7 @@ This library requires Apache Spark 2.0+ and Amazon Redshift 1.0.963+.
 
 For version that works with Spark 1.x, please check for the [1.x branch](https://github.com/databricks/spark-redshift/tree/branch-1.x).
 
-You may use this library in your applications with the following dependency information:
-
-**Scala 2.10**
-
-```
-groupId: com.databricks
-artifactId: spark-redshift_2.10
-version: 3.0.0-preview1
-```
-
-**Scala 2.11**
-```
-groupId: com.databricks
-artifactId: spark-redshift_2.11
-version: 3.0.0-preview1
-```
-
-You will also need to provide a JDBC driver that is compatible with Redshift. Amazon recommend that you use [their driver](http://docs.aws.amazon.com/redshift/latest/mgmt/configure-jdbc-connection.html), which is distributed as a JAR that is hosted on Amazon's website. This library has also been successfully tested using the Postgres JDBC driver.
```

Comment:
Why did you move this down ?

Reply:
I merged the snapshot section with the release section because there are no releases (ie. master-SNAPSHOT),

Reply:
See above

Reply:
yup